src/pseudopeople/interface.py

```diff
@@ -6,22 +6,30 @@
 
 from pseudopeople.entities import Form
 from pseudopeople.noise import noise_form
-from pseudopeople.utilities import get_default_configuration
+from pseudopeople.utilities import (
+    get_default_configuration,
+    update_configuration_with_yaml,
+)
 
 
 # TODO: add year as parameter to select the year of the decennial census to generate (MIC-3909)
 # TODO: add default path: have the package install the small data in a known location and then
 #  to make this parameter optional, with the default being the location of the small data that
 #  is installed with the package (MIC-3884)
-def generate_decennial_census(path: Union[Path, str], seed: int = 0):
+def generate_decennial_census(
+    path: Union[Path, str], seed: int = 0, override: Union[Path, str] = None
```

Comment:
Get feedback from the RT regarding the argument name `override`.

Reply:
I've asked in Slack. I'll update with any changes.